docs/reference/multisite.rst

```diff
@@ -100,38 +80,16 @@ The last action is to configure the ``sonata_page`` section as:
 Host and Path By Locale Strategy
 --------------------------------
 
-This strategy requires a dedicated ``RequestFactory`` object. So you need to alter the
-front controller to use the one provided by the ``PageBundle``.
-
-To do so, open ``index.php`` file and change the following parts::
-
-    // public/index.php
-
-    use App\Kernel;
-    use Symfony\Component\ErrorHandler\Debug;
-    use Sonata\PageBundle\Request\RequestFactory; # before: use Symfony\Component\HttpFoundation\Request;
-
-    require dirname(__DIR__).'/config/bootstrap.php';
-
-    if ($_SERVER['APP_DEBUG']) {
-        umask(0000);
-
-        Debug::enable();
-    }
+This strategy requires a dedicated runtime. So you need to modify your
```

Comment:
I guess we could symplify this doc saying

In case the user wants to use `Host path` or `Host path by locale`, then add the runtime.